python/ml4ir/applications/ranking/model/losses/listwise_losses.py

```diff
@@ -91,6 +91,99 @@ def final_activation_op(self, inputs, training=None):
         return self.final_activation_fn(logits)
 
 
+class AuxiliaryOneHotCrossEntropy(SoftmaxCrossEntropy):
```

Comment:
Both the aux losses are inherited from `SoftmaxCrossEntropy` because they are using the same activation function on `y_pred` / logits.